corehq/apps/app_manager/views/form_designer.py

```diff
@@ -76,6 +76,19 @@ def form_designer(request, domain, app_id, module_id=None, form_id=None,
         'custom_intents': domain_has_privilege(domain, privileges.CUSTOM_INTENTS),
     })
 
+    scheduler_data_nodes = []
+    if getattr(module, 'has_schedule', False):
+        scheduler_data_nodes = [
+            "unscheduled_visit",
+            "current_visit_number",
+            "next_visit",
+            "next_visit_date",
+            "next_due",
+        ]
```

Comment:
@czue Yes, these probably should have been namespaced in a group. I'll review with Matt next week when I'm back from vacation to see how much of a pain it is going to be to change the API for the ICDS app they've built.

@emord, all of these are in `app_manager/const.py`, you may want to use those definitions instead (but yes, that is the whole list). Also, each advanced form has a `schedule.enabled` property, which should be coupled with `module.has_schedule` to check if the schedule is active on that particular form: 
```py
has_schedule = (module.has_schedule and getattr(form, 'schedule', False) and form.schedule.enabled)
```